### ISSUE TITLE
failed to find "useMeta" imported from "#imports"   nuxt3-3.0.0

### ISSUE THREAD

**Issue by impohcom** [vueuse/vueuse#2453]
### Describe the bug

failed to find "useMeta" imported from "#imports"  

### Reproduction

[vite-node] [plugin:nuxt:imports-transform] [VITE_ERROR] /node_modules/@vueuse/nuxt/ssr-plugin.mjs

### System Info

```Shell
[vite-node] [plugin:nuxt:imports-transform] [VITE_ERROR] /node_modules/@vueuse/nuxt/ssr-plugin.mjs

import { setSSRHandler } from '@vueuse/core'
import { defineNuxtPlugin, useCookie, useMeta } from '#imports'
```

### Used Package Manager

pnpm

### Validations

- [x] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [x] Read the [docs](https://vueuse.org/guide).
- [x] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by impohcom** [comment]
[[[我相信 useMeta 已被 useHead
[SEO 和 Meta取代](https://nuxt.com/docs/getting-started/seo-meta/)](https://github.com/nuxt/framework/issues/9105#issuecomment-1318357149)]

**Comment by shtse8** [comment]
This should be fixed in this (PR)[https://github.com/vueuse/vueuse/pull/2449]

**Comment by webfansplz** [comment]
Fixed in v9.6.0
